src/app/admin/auditLogTable/types.ts

```diff
@@ -16,16 +19,20 @@ export interface AuditLogRow {
     parcelId: string;
     profileId: string;
     statusOrder: string;
-    wasSuccess: boolean;
+    wasSuccess: boolean | null;
     websiteData: string;
 }
 
 export const convertAuditLogResponseToAuditLogRow = (
-    auditLogResponse: Schema["audit_log"][]
+    auditLogResponse: AuditLogResponseData
 ): AuditLogRow[] =>
     auditLogResponse.map((auditLogResponseRow) => ({
+        auditLogId: auditLogResponseRow.primary_key ?? "",
         action: auditLogResponseRow.action ?? "",
-        actorProfileId: auditLogResponseRow.actor_profile_id ?? "",
+        actorName:
+            auditLogResponseRow.actor_user_id === null
+                ? "User Deleted"
```

Comment:
to be consistent with the modal row (or even put this in a common const)
```suggestion
                ? "Deleted User"
```